README.md

```diff
@@ -5,6 +5,10 @@
 Middleware for express to enable opentracing.
 Supports any opentracing tracer compatible with version 0.11.0 of the opentracing javascript library.
 
+## Migration
+
+Please note that this library has been moved to [https://github.com/opentracing-contrib](https://github.com/opentracing-contrib). Please use that version instead.
```

Comment:
🙇 can we link to https://github.com/opentracing-contrib/javascript-express directly.

Reply:
Thanks, hadn't noticed that I didn't. I've done that in 125b2f0